latest/index.bs

```diff
@@ -175,7 +175,294 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+        |                     # `.zattrs` the "intensity_image" MUST be in the same coordinate system as the label image specified in "region".
+        │
+        ├── X                 # You MAY add an zarr array `X`.
+        │   │                 # `X` MUST not be a complex type (i.e., MUST be a single type)
+        │   │                 # `X` MAY be chunked as the user desires.
+        │   ├── .zarray
+        │   ├── 0.0
+        │   │   ...
+        │   └── n.m
+        |
+        ├── layers            # You MAY add a `layers` group, which contains dense matrices with the same shape as X.
+        │   │
+        │   ├── .zgroup
+        │   ├── .zattrs
+        │   │
+        │   └── layer_0       # You MAY add a zarr array for each layer
+        |       |             # Each layer array MUST have the same shape as X
+        |       |             # Each layer array SHOULD be chunked the same as X
+        |       ├── .zarray
+        |       |
+        |       ├── 0.0
+        │       │   ...
+        │       └── n.m
+        │        
+        ├── obs               # You MUST add an obs group container. The obs group holds a table of annotations on the rows in X.
+        │   │                 # The rows in obs MUST be index-matched to the rows in X.
+        │   ├── .zgroup
+        │   │                     
+        │   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
+        │   │                 # `.zattrs` MUST contain `"column-order"`, which is a list of the order of the non-_index columns.
+        │   │                 # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dataframe"` by AnnData.
+        │   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.2.0"` by AnnData.
+        │   │      
+        │   └── col_0         # Each column in the obs table is a 1D zarr array. The rows can be chunked as the user desires.
+        │       ├── .zarray   # However, the obs columns SHOULD be chunked in the same way as the rows in X (if present).
+        │       │
+        │       └─ 0
+        ├── var               # You MAY add a var group container. The var group holds a table of annotations on the columns in X.
+        |   │                 # The rows in var MUST be index-matched to the columns in X (if present). 
+        |   |
+        |   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
+        |   │                 # `.zattrs` MUST contain `"column-order"`, which is a list of the order of the non-_index columns.
+        |   │                 # `.zattrs` MUST contain `"encoding-type"`, which is set to `"dataframe"` by AnnData.
+        |   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
```

Comment:
```suggestion
        |   │                 # `.zattrs` MUST contain `"encoding-version"`, which is set to `"0.2.0"` by AnnData.
```

The specification for dataframes is repeated multiple places, which is leading to errors. It would be good for these to be factored out so they aren't repeated.